packages/fiori-elements-writer/src/data/annotationReuseLibs.ts

```diff
@@ -0,0 +1,58 @@
+import { OdataVersion } from '../types';
+
+interface AnnotationReuseLibsEntry {
+    annotation: string;
+    reuseLib: string;
+}
+
+type AnnotationReuseLibs = {
+    [V in OdataVersion]: [AnnotationReuseLibsEntry] | [];
+};
+
+export const annotationReuseLibs: AnnotationReuseLibs = {
+    [OdataVersion.v2]: [],
+    [OdataVersion.v4]: [
+        {
+            annotation: 'UI.Note',
+            reuseLib: 'sap.nw.core.gbt.notes.lib.reuse'
+        }
+    ]
+};
+
+/**
+ * Returns the reuse libraries associated with annotation entries in the metadata
+ *
+ * @param version - the odata service version 
+ * @param metadata - metadata string to be checked for specific annotations
+ * @returns The base component library path
+ */
+export function getAnnotationLibs(version: OdataVersion, metadata?: string) {
```

Comment:
```suggestion
export function getAnnotationLibs(version: OdataVersion, metadata: string) {
```
`metadata` should be mandatory to read the annotations

Reply:
Thanks, I have added your suggestion.